campaignion_foundation.js

```diff
@@ -59,16 +65,20 @@ Drupal.behaviors.campaignion_foundation_clientside_validation.attach = function
 
   // Add classes to form elements.
   $('form', context).on('clientsideValidationInvalid', function (event, element) {
-    var $wrapper = findWrapper($(element));
-    $(element).addClass('is-invalid-input');
-    $wrapper.siblings('label').addClass('is-invalid-label');
-    $wrapper.next('.form-error').addClass('is-visible');
+    if (element) {
+      var $wrapper = findWrapper($(element));
+      $(element).addClass('is-invalid-input');
+      $wrapper.siblings('label').addClass('is-invalid-label');
+      $wrapper.siblings(`.form-error[for=${element.id}]`).addClass('is-visible');
```

Comment:
Maybe we shouldn’t use es6 template literals, IE11 and Opera Mini still don’t support them…